docs/docs/reference-react-component.md

```diff
@@ -44,27 +99,47 @@ constructor(props) {
 
 ### `.componentDidMount()`
 
-TODO
+`componentDidMount()` is invoked immediately after a component is mounted. Initialization that requires DOM nodes should go here. If you need to load data from a remote endpoint, this is a good place to instantiate the network request. Setting state in this method will trigger a re-rendering.
 
 ### `.componentDidUpdate(prevProps, prevState)`
 
-TODO
+`componentDidUpdate()` is invoked immediately after updating occurs. This method is not called for the initial render.
+
+Use this as an opportunity to operate on the DOM when the component has been updated.
```

Comment:
This is also a good place to do network requests as long as you compare the props to previous props.